### DISCUSSION TITLE
Auto delete releases in history list

### DISCUSSION THREAD

**Discussion by blixten85** [autobrr/autobrr#1817] [Ideas]
I would like to have some options to automatically delete releases in the history list that are older than x amount of time. 

Personally I was thinking a week or two. But some pre-made options with the possibility to add a custom timer would be perfect. 

I also want to keep all releases that has been approved regardless of age. But that kind of customisation is already possible. Just not automatically yet!

**Comment by mrgeetv** [comment]
This functionality has been released. See https://github.com/autobrr/autobrr/releases/tag/v1.72.0 and https://github.com/autobrr/autobrr/pull/2247.
